test/fixtures/pretty-print.json

```diff
@@ -1,14 +1,14 @@
 {
   "chalk": {
-    "replace": true,
+    "npm": true,
```

Comment:
please keep the tests in their original state. Only change if it is for an explicit reason